### DISCUSSION TITLE
Energy Meter

### DISCUSSION THREAD

**Discussion by TimothyClose** [arachnetech/homebridge-mqttthing#582] [General]
Hi ,

I am trying to setup an energy meter and don't see anything similar on the lists. Is there any advice on how to tackle this.

the reading that are available are :

wem3080_voltage | V | Voltage.
wem3080_current | A | current.
wem3080_power | W | active power.
wem3080_importenergy | kWh | Energy consumption from grid
wem3080_exportgrid | kWh | Energy export to grid

Hoping for any advice

**Comment by HolgerAusB** [comment]
Have a look to Outlets. They have  fields for V, A, W and one kWh
